content/news/2022-09-02-gsoc-final-blog-david-chocholaty.md

```diff
@@ -642,7 +676,9 @@ and pull requests. Thank you both for involving me in the Mixxx development
 process and for constructive criticism which offers me learn many new things
 in the past weeks. I would like to thank my summer colleague for the Mixxx
 organization and friend [@Fatih Emre](https://github.com/fatihemreyildiz)
-for his help and synergy on the final blog structure and chapters. Of course,
+for his help and synergy on the final blog structure and chapters. Thank you
+to [@Be.](http://127.0.0.1:8000/news/author/be) for taking care of publishing
```

Comment:
```suggestion
to [@Be.]({author}be) for taking care of publishing
```

Reply:
You made the video. Uploading it was trivial. You don't need to credit me for that.

Reply:
Right :smile:

Reply:
I'd prefer to not be credited here.

Reply:
Okay, no problem. I will fix it.